ext/Hash-Util/lib/Hash/Util.pm

```diff
@@ -509,13 +509,19 @@ Perl has been built with. Possible sizes may be but are not limited to
 =item B<hash_value>
 
     my $hash_value = hash_value($string);
-
-hash_value() returns the current perl's internal hash value for a given
-string.
-
-Returns a 32 bit integer representing the hash value of the string passed
-in. This value is only reliable for the lifetime of the process. It may
-be different depending on invocation, environment variables,  perl version,
+    my $hash_value = hash_value($string, $seed);
+
+hash_value($string)
+returns the current perl's internal hash value for a given string.
```

Comment:
POD formatting recommendation:  Enclose `hash_value($string)` in C<...> code formatting.